docs/service-introspection-onboarding.md

```diff
@@ -125,10 +125,13 @@ Paste the following task in its corresponding `azure-pipelines.yml`:
     commitId=$(echo "${commitId:0:7}")
     service=$(Build.Repository.Name)
     service=${service##*/}
-    VERSION_TO_DOWNLOAD=$(curl -s "https://api.github.com/repos/CatalystCode/spk/releases/latest" | grep "tag_name" | sed -E 's/.*"([^"]+)".*/\1/')
-    echo "Downloading SPK version $VERSION_TO_DOWNLOAD" && wget "https://github.com/CatalystCode/spk/releases/download/$VERSION_TO_DOWNLOAD/spk-linux"
-    chmod +x ./spk-linux
-    ./spk-linux deployment create -n $(ACCOUNT_NAME) -k $(ACCOUNT_KEY) -t $(TABLE_NAME) -p $(PARTITION_KEY) --p1 $(Build.BuildId) --image-tag $tag_name --commit-id $commitId --service $service
+    echo "Downloading the latest SPK version"
```

Comment:
The download script in bedrock uses a variable to determine if a specific version is specified to use spk. Maybe the wording should say "Downloading SPK" to be less specific?

Reply:
Are you suggesting I remove "latest SPK version" from that log text?

Reply:
Yup :)